exercises/Toluwalase_Agoro/ex01_basics/src/main.cpp

```diff
@@ -0,0 +1,184 @@
+
+#include <iostream>
+#include <vector>
+#include <fstream>
+#include <string>
+#include <iomanip>
+using namespace std;
+
+
+void toLower(string &stringName) {
+	/* Function accepts a string and modifies string
+	by making it lowercase and returns nothing
+	*/
+	string PlaceHolderString{  };
+	for (char Letter : stringName) 
```

Comment:
Two rather pedantic points relating to this range based for loop:
- we tend to use auto in these ie for (auto Letter : stringName). If you've not come across the auto keyword let's discuss
- where the iterator isn't being changed we normally use & ie for (auto& Letter : stringName)

In this example both changes will have minimal functional impact but in a different case where the iterator is a bigger class then it would improve performance.